lib/Controller/RoomController.php

```diff
@@ -137,13 +138,17 @@ public function update(
 			return new DataResponse(['message' => 'Not allowed to enable recordings.'], Http::STATUS_BAD_REQUEST);
 		}
 
+		if (!$restriction->getallowLogoutURL() && $logoutURL !== $room->getlogoutURL()) {
```

Comment:
Is the value of `allowLogoutURL` determined somewhere? You should probably look into the `Permission` class :wink:

Reply:
The value of `allowLogoutURL` is determined in the `Restriction` class, together with the other restrictions.